ddtrace/internal/writer/writer.py

```diff
@@ -276,16 +265,14 @@ def _put(self, data, headers, client, no_trace):
                 if not self._reuse_connections:
                     self._reset_connection()
 
-    def _get_finalized_headers(self, count, client):
-        # type: (int, WriterClientBase) -> dict
+    def _get_finalized_headers(self, count: int, client: WriterClientBase) -> dict:
         headers = self._headers.copy()
         headers.update({"Content-Type": client.encoder.content_type})  # type: ignore[attr-defined]
         if hasattr(client, "_headers"):
             headers.update(client._headers)
         return headers
 
-    def _send_payload(self, payload, count, client):
-        # type: (...) -> Response
+    def _send_payload(self, payload, count, client) -> Response:
```

Comment:
Should we type hint these too?
```suggestion
    def _send_payload(self, payload: bytes, count: int, client: WriterClientBase) -> Response:
```